internal/cluster/clusterdriver/transport_http.go

```diff
@@ -145,8 +145,8 @@ func getOrgID(req *http.Request) (uint, error) {
 }
 
 func encodeDeleteClusterHTTPResponse(_ context.Context, w http.ResponseWriter, response interface{}) error {
-	resp, ok := response.(DeleteClusterResponse)
-	if !ok || resp.Deleted {
+	_, ok := response.(DeleteClusterResponse)
+	if !ok {
```

Comment:
**Question**: is the `deleted` flag used when the cluster is still in deleting state?
(I'm arguing myself whether this is a good thing or not, I get it we mean the cluster has started to be deleted, but I want to double check whether the post-delete behavior changes because of this, I would think so.)